src/Core/Rexl.Bind/Strings/RexlStrings.tt

```diff
@@ -336,6 +336,10 @@ AboutLower_Source = "The text to convert to lowercase."
 AboutUpper = "Converts text to uppercase."
 AboutUpper_Source = "The text to convert to uppercase."
 
+AboutPadLeft = "Adds characters to the left of a string."
```

Comment:
Should mention that it adds spaces. Also mention that it adds just enough spaces so the result is at least the length specified.